docs/experimental_grpc_python_api.rst

```diff
@@ -35,13 +35,13 @@ This module contains most of the core functionality of the library
 including setting up a connection, sending requests and receiving
 response to/from an active Triton server.
 
-.. automodule:: tritongrpcclient.core
+.. automodule:: tritongrpcclient.grpcclient
```

Comment:
The naming seems unnecessarily redundant. Is this the pythonic way? What is we just use __init__ for the grpcclient.py and then we would just need:

import tritongrpcclient